ff/serving/runner/kubernetes.go

```diff
@@ -121,7 +127,14 @@ func (k KubernetesRunner) Run() (CompletionWatcher, error) {
 	return KubernetesCompletionWatcher{jobClient: k.jobClient}, nil
 }
 
-func NewKubernetesRunner(config KubernetesRunnerConfig) (Runner, error) {
+func (k KubernetesRunner) Schedule(schedule string) error {
```

Comment:
Lets make the string a struct with similar fields to the string so its easy to implement for other things